spec/features/dashboard_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+feature 'dashboard' do
+  context 'authorized user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.